docs/src/getting_started.md

```diff
@@ -11,7 +11,18 @@ using Pkg
 Pkg.add(ClimaLand)
 ```
 
-## Running a simple soil model
+## Steps to run a model
+
+To run a model, you will always need to do these steps:
+- Import dependencies. You need some libraries to run the model.
+- Chose Float32 or Float64.
+- Import parameters. You need some constants and parameters values to run the model.
+- Define your domains. You need a domain to solve PDEs. See the [tutorial to ClimaLand domains!](@ref "Domain Tutorial")
+- Set your simulation period: start date, end date, timestep.
+- Import the forcings (for example, air temperature, precipitations).
```

Comment:
maybe "Set up the forcing data" since "Import" has a specific meaning, and the forcings are not a module the user imports.